### ISSUE TITLE
Escape key doesn't work on wayland

### ISSUE THREAD

**Issue by UtkarshVerma** [slgobinath/SafeEyes#480]
**Describe the bug**
Pressing escape during a break doesn't exit safeeyes.

**To Reproduce**
1. Take a break
2. Press escape
3. Nothing happens

**Expected behavior**
Safeeyes break should be stopped.

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Desktop Env: SwayWM / Wayland
 - Version: 2.1.3

**Comment by archisman-panigrahi** [comment]
Given that we currently have a setting that does not allow breaks to be skipped, do we really want to add escape to skip a break? That would render https://github.com/slgobinath/SafeEyes/pull/603 kind of useless

**Comment by deltragon** [comment]
@archisman-panigrahi This is referring to a feature that already exists on X11 - there, pressing Escape on the Break Screen does the same thing as clicking the "Skip" button. It also respects the setting from #603.
This issue is just complaining that Wayland is regressing here compared to X11.

**Comment by archisman-panigrahi** [comment]
I cannot reproduce with Arch Linux KDE 6.1.3 + Wayland + Safe Eyes 2.2.1.
The escape key closes the break screen.

If anyone can reproduce the issue, please reopen.

**Comment by deltragon** [comment]
Oh, that's surprising to me - I could have sworn I managed to reproduce this one.
I'll check tomorrow.

**Comment by archisman-panigrahi** [comment]
Is the intended feature something like this?

When skip button is shown, escape key plays the same role as the skip button.
When skip button is hidden, escape key does not do anything.

**Comment by deltragon** [comment]
Yes, this is the intended feature. This is also how it works on X11 (apart from some bugs).
